### PR TITLE
Add resnext50_32x4d

### PR DESCRIPTION
**Pull request by barrh**
This patch adds the pretrained network ResNext50, that I ported with the recommended package (https://github.com/clcarwin/convert_torch_to_pytorch)
Due to its size, the pytorch model itself isn't part of the patch, but I'll be happy to deliver it separately. I left a TODO in the patch concerning the URL.

The ported model has slightly lower top1 accuracy than the paper: 77.1 (vs. 77.8).

I didn't modify the README file.

### CODE REVIEW

**Comment by barrh** [comment]
Model is added to torchvision.
